### PR TITLE
 Issue Template Updates for Figma link added #391

### LINKED ISSUE #391: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by csengineer1990**
**Description**
Current Behavior

The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

Desired Situation
Each template that has a reference to Figma in its resources section should have an invite link added.

This PR fixes #391
Figma invite link added.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
 ---
 name: Documentation issue
 about: Issues related to documentation.
-title: '[Docs]'
-labels: 'docs'
-assignees: ''
+title: "[Docs]"
+labels: "docs"
+assignees: ""
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/documentation.md:6]: Please revert this change

Comment (csengineer1990) [reply .github/ISSUE_TEMPLATE/documentation.md]: changes done. please check if anything wrong.

**Comment by sudhanshutech** [comment]
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by csengineer1990** [comment]
> Let's discuss it on the website call. Please add this as an agenda item in the meeting minutes, if you would :)
> 
> https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima


Sir @sudhanshutech I am a newcomer and I don't know how to add an agenda what will I talk about it in the meeting.?

**Comment by csengineer1990** [comment]
> @csengineer1990, thanks much, please update rest of the issue templates too

I have made the changes please check and please let me know if anything wrong?

**Review by abhijeetgauravm — COMMENTED** [review]
hey @csengineer1990 similarly update other files of issue template with the provided figma links!

**Review by abhijeetgauravm — CHANGES_REQUESTED** [review]
hey @csengineer1990 replace the lines that has a reference to Figma . Right now u added the links again , just replace with the lines which have a reference to Figma .

**Comment by csengineer1990** [comment]
> hey @csengineer1990 replace the lines that has a reference to Figma . Right now u added the links again , just replace with the lines which have a reference to Figma because they are missing an open invite link.

ok, sir. I replace it. please check.

**Review by abhijeetgauravm — APPROVED** [review]
hey @csengineer1990 also do the changes in bug_report.md files.

**Review by leecalcote — APPROVED** [review]
Thanks!
